File: Beecrowd/Python/1019 - Time Conversion.py
time = int(input())

if time>=3600:
    h = time//3600;
    time = time%3600
    
    print(f"{h}:",end="")
else:
    print(f"0:",end="")
    
if(time>=60):
    m = time//60;
    time = time%60
    print(f"{m}:{time}")
else:
     print(f"0:{time}")

File: Beecrowd/C/1019 - Time Conversion.c
                                                                           
#include <stdio.h>

int main()
{
    int time;
        
    scanf("%d", &time);
        
    int hours = time / 3600;
    time = time - (hours * 3600);
        
    int minutes = time / 60;
    time = time - (minutes * 60);
        
    printf("%d:%d:%d\n", hours, minutes, time);
        
 return 0;
}